src/components-examples/material/table/table-multiple-row-template/table-multiple-row-template-example.html

```diff
@@ -0,0 +1,38 @@
+<div class="mat-elevation-z8">
+    <table mat-table [dataSource]="dataSource" multiTemplateDataRows>
+    
+        <!-- Position Column -->
+        <ng-container matColumnDef="position">
+        <th mat-header-cell *matHeaderCellDef> No. </th>
```

Comment:
Indentation in this file and the .css one above looks off. It's probably using tabs instead of spaces.